eventstore/tests/test_views.py

```diff
@@ -1390,19 +1395,21 @@ def test_successful_outbound_messages_request(self, mock_handle_outbound):
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         [messages] = Message.objects.all()
         self.assertEqual(str(messages.contact_id), "whatsapp-id")
-        self.assertEqual(messages.id, "message-id"),
-        self.assertEqual(messages.type, "text"),
-        self.assertEqual(messages.message_direction, Message.OUTBOUND),
-        self.assertEqual(
-            messages.data,
-            {
-                "text": {"body": "your-text-message-content"},
-                "render_mentions": True,
-                "preview_url": True,
-                "random": "data",
-                "recipient_type": "individual",
-            },
-        ),
+        (self.assertEqual(messages.id, "message-id"),)
+        (self.assertEqual(messages.type, "text"),)
+        (self.assertEqual(messages.message_direction, Message.OUTBOUND),)
+        (
+            self.assertEqual(
+                messages.data,
+                {
+                    "text": {"body": "your-text-message-content"},
+                    "render_mentions": True,
+                    "preview_url": True,
+                    "random": "data",
+                    "recipient_type": "individual",
+                },
+            ),
+        )
```

Comment:
I think we can just remove the trailing comma that was probably added as a mistake. They don't need to be tuples

Reply:
👍